docs/dev-tools/zombienet/overview.md

```diff
@@ -235,417 +243,319 @@ Zombienet provides a CLI that allows interaction with the tool. The CLI can rece
 
-|                   Flag                    |                                                Description                                                |
-| :---------------------------------------: | :-------------------------------------------------------------------------------------------------------: |
-|            `-p`, `--provider`             |  Override provider to use (choices: `podman`, `default`, and, `native`). By default it uses `kubernetes`  |
-|           `-d`, `--dir` <path>            | Directory path for placing the network files instead of random temp one (e.g. -d /home/user/my-zombienet) |
-|              `-f`, `--force`              |                                    Force override all prompt commands                                     |
-|        `-l`, `--logType` <logType>        |   Type of logging on the console (choices: `table`, `text`, and, `silent`). By default it uses `table`    |
-|             `-m`, `--monitor`             |                              Start as monitor, do not auto clean up network                               |
-| `-c`, `--spawn-concurrency` <concurrency> |                   Number of concurrent spawning process to launch. By default it is `1`                   |
-|              `-h`, `--help`               |                                         Display help for command                                          |
+??? function "`setup` - set up the Zombienet development environment"
+
+    === "Argument"
+
+        - `<binaries>` - executables that will be downloaded and prepared to be used by Zombienet. Options: `polkadot`, `polkadot-parachain`
+
+??? function "`convert` - transforms a (now deprecated) polkadot-launch configuration file to a Zombienet configuration file"
+
+    === "Argument"
+
+        - `<filePath>` - path to a [polkadot-launch](https://github.com/paritytech/polkadot-launch){target=_blank} configuration file with a `.js` or `.json` extension defined by [the `LaunchConfig` interface](https://github.com/paritytech/polkadot-launch/blob/295a6870dd363b0b0108e745887f51e7141d7b5f/src/types.d.ts#L10){target=_blank}
+
+??? function "`version` - prints Zombienet version"
+
+    === "Argument"
+
+        None 
+
+??? function "`help` - prints help information"
+
+    === "Argument"
+
+        None 
+
+### CLI Flags
+
+You can use the following flags to customize the behavior of the CLI:
+
+??? function "`-p`, `--provider` - override provider to use. Defaults to `kubernetes`"
+    
+    === "Argument"
 
+        - `<provider>` - the provider to use. Options: `podman`, `kubernetes`, `native`
+
+??? function "`-d`, `--dir` - directory path for placing the network files instead of random temp one"
+
+    === "Argument"
+
+        - `<path>` - desired path for network files  
+
+    === "Example"
+
+        - `-d /home/user/my-zombienet`
```

Comment:
```suggestion
        `zombienet -d /home/user/my-zombienet`
```

This doesn't need to be bulleted since it's code, and let's use 3 back ticks so users can copy the example if they need to